### ISSUE TITLE
Dotted lines that separate "fast-average-slow" areas are not in place.

### ISSUE THREAD

**Issue by opusdei** [edgemesh/crux.run#4]
I think that the lines that separate the three areas: fast, average and slow are not drawn at the correct points. Attached is an image of the result, and of what it should be, according to the data. As you can see, there are "fast loading" time points within the "average loading time" range; and "average" points in the "slow" range.

Current drawing:

![imagen](https://user-images.githubusercontent.com/5907868/64520512-9f6cb480-d2f6-11e9-9c6f-028f8032cc1b.png)

Corrected drawing:

![imagen](https://user-images.githubusercontent.com/5907868/64521174-cd9ec400-d2f7-11e9-81f7-4b422916719d.png)

**Comment by jacobloveless** [comment]
@opusdei  Thanks for reporting this!

**Comment by yrokhlin** [comment]
So originally these ref lines were really only meant to be just a quick relative guide, but I can see where it would cause confusion. I removed the old reference lines and I have replaced them with axis specific notation lines, that will always point to the correct fast/slow values.

![image](https://user-images.githubusercontent.com/6445303/64580462-4115f500-d33b-11e9-9a7a-514528021f59.png)

Also, the gradient should now correspond correctly to the actual value of the histogram, so it should now be functional instead of just aesthetic.

![image](https://user-images.githubusercontent.com/6445303/64580699-1f693d80-d33c-11e9-81b2-9bb3c586be68.png)


This should be live now on https://crux.run (although cache might take a few minutes)
